addons/project/static/src/views/project_task_form/project_task_form_controller.js

```diff
@@ -1,11 +1,17 @@
 /** @odoo-module */
 
 import { FormController } from '@web/views/form/form_controller';
-import { DeleteSubtasksConfirmationDialog } from "@project/components/delete_subtasks_confirmation_dialog/delete_subtasks_confirmation_dialog";
 import { _t } from "@web/core/l10n/translation";
 import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
 import HistoryDialog from '@web_editor/components/history_dialog/history_dialog';
 
+export const subTaskDeleteConfirmationMessage = _t(
+    `Deleting a task will also delete its associated sub-tasks. \
```

Comment:
The backslash is really needed?

Reply:
This is because of the linter you have to do this?

Reply:
Yes exactly because of the linter I have to do this, otherwise we can eliminate it and add it in a single line.

Reply:
Okay, I see. 🙂